worlds/timespinner/Options.py

```diff
@@ -388,67 +357,256 @@ class TrapChance(Range):
+            self.fixed_drop_rate.value = self.FixedDropRate.value
+            has_replaced_options = True
+        if self.LootTierDistro != LootTierDistro.default and \
+            self.loot_tier_distro == LootTierDistro.default:
+            self.loot_tier_distro.value = self.LootTierDistro.value
+            has_replaced_options = True
+        if self.ShowBestiary != ShowBestiary.default and \
+            self.show_bestiary == ShowBestiary.default:
+            self.show_bestiary.value = self.ShowBestiary.value
+            has_replaced_options = True
+        if self.ShowDrops != ShowDrops.default and \
+            self.show_drops == ShowDrops.default:
+            self.show_drops.value = self.ShowDrops.value
+            has_replaced_options = True
+        if self.EnterSandman != EnterSandman.default and \
+            self.enter_sandman == EnterSandman.default:
+            self.enter_sandman.value = self.EnterSandman.value
+            has_replaced_options = True
+        if self.DadPercent != DadPercent.default and \
+            self.dad_percent == DadPercent.default:
+            self.dad_percent.value = self.DadPercent.value
+            has_replaced_options = True
+        if self.RisingTides != RisingTides.default and \
+            self.rising_tides == RisingTides.default:
+            self.rising_tides.value = self.RisingTides.value
+            has_replaced_options = True
+        if self.RisingTidesOverrides != RisingTidesOverrides.default and \
+            self.rising_tides_overrides == RisingTidesOverrides.default:
+            self.rising_tides_overrides.value = self.RisingTidesOverrides.value
+            has_replaced_options = True
+        if self.UnchainedKeys != UnchainedKeys.default and \
+            self.unchained_keys == UnchainedKeys.default:
+            self.unchained_keys.value = self.UnchainedKeys.value
+            has_replaced_options = True
+        if self.PresentAccessWithWheelAndSpindle != PresentAccessWithWheelAndSpindle.default and \
+            self.back_to_the_future == PresentAccessWithWheelAndSpindle.default:
+            self.back_to_the_future.value = self.PresentAccessWithWheelAndSpindle.value
+            has_replaced_options = True
+        if self.TrapChance != TrapChance.default and \
+            self.trap_chance == TrapChance.default:
+            self.trap_chance.value = self.TrapChance.value
+            has_replaced_options = True
+        if self.Traps != Traps.default and \
+            self.traps == Traps.default:
+            self.traps.value = self.Traps.value
+            has_replaced_options = True
+        if self.DeathLink != DeathLink.default and \
+            self.death_link == DeathLink.default:
+            self.death_link.value = self.DeathLink.value
+            has_replaced_options = True
+
+        if has_replaced_options:
+            logging.warning("Timespinner options where renamed from PasCalCase to snake_case, plz update your yml")
```

Comment:
End of File newline, also it's far more commonly "yaml" than "yml"
```suggestion
            logging.warning("Timespinner options where renamed from PasCalCase to snake_case, plz update your yaml")
            
```